tests/utils/tools.php

```diff
@@ -45,7 +45,7 @@ function drop_collection($uri, $databaseName, $collectionName)
     $command = new Command(['drop' => $collectionName]);
 
     try {
-        $server->executeCommand($databaseName, $command);
+        $server->executeCommand($databaseName, $command, [ 'writeConcern' => new \MongoDB\Driver\WriteConcern( \MongoDB\Driver\WriteConcern::MAJORITY ) ] );
```

Comment:
Why was this necessary? Did it interfere with tests that query the secondary and expected an empty collection?

Reply:
No: https://jira.mongodb.org/browse/SERVER-35613 — I need to come up with whether I think this is Okay behaviour in the server though. It's certainly going to be mega confusing for users.

Reply:
Reminder to add a comment and reference to SERVER-35613.